Add per-method options for MoleClient, maxPacketSize for MoleServer and better error handling

artemtrush opened the pull request:
Check CHANGELOG for details

artemtrush committed:
Add single-method timeout for MoleClient and maxPacketSize for MoleServer

artemtrush committed:
Fix undefined variable

artemtrush committed:
Add better error message

artemtrush committed:
Add index.js file

artemtrush committed:
Update linux dist for Travis

artemtrush committed:
Merge pull request #1 from Delos-tech/feature/mole-options

Add single-method timeout for MoleClient and maxPacketSize for MoleServer

artemtrush committed:
Add shutdown method for client and server

artemtrush committed:
Merge pull request #2 from Delos-tech/feature/shutdown

Add shutdown method for client and server

koorchik commented on CHANGELOG.md:
```diff
@@ -0,0 +1,19 @@
+## [1.2.0] - 2024-03-22
+
+### Added
+- options parameter for `MoleClient.callMethod` to pass `timeout` to it
```
Could you update Readme accordingly? 
* Add timeout option description
*  Add shutdown method description

artemtrush replied:
Yep

artemtrush committed:
Merge branch 'koorchik:master' into master

artemtrush committed:
Update mole-rpc documentation

samamos committed:
Create stale-issues-and-prs.yml

Adds stale issue and pr workflow per [CUBE-1633](https://delosliving.atlassian.net/browse/CUBE-1633)

[CUBE-1633]: https://delosliving.atlassian.net/browse/CUBE-1633?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

samamos committed:
Merge pull request #3 from Delos-tech/CUBE-1633-stale-automation

Create stale-issues-and-prs.yml
